distribution/lib/Standard/Base/0.0.0-dev/src/Network/HTTP/Response.enso

```diff
@@ -30,6 +40,13 @@ type Response
         header_entries = Vector.from_polyglot_array (Http_Utils.get_headers self.internal_http_response.headers)
         header_entries.map e-> Header.new e.getKey e.getValue
 
+    ## Get the response content type.
+    content_type : Text | Nothing
+    content_type self =
+        response_headers = self.headers
+        content_type = response_headers.find if_missing=Nothing h-> "Content-Type".equals_ignore_case h.name
```

Comment:
Maybe that name would be clearer?

```suggestion
        content_type_header = response_headers.find if_missing=Nothing h-> "Content-Type".equals_ignore_case h.name
```

Reply:
Also, that is a pretty tedious way to figure out the content type.

I guess its simple code and probably not a preformance intensive place, so we can leave as-is.

But instead of constructing the whole Header Map in Enso and copying all headers around, we could just use the Java object directly:
```suggestion
        java_optional = self.internal_http_response.headers.firstValue "Content-Type"
        if java_optional.isPresent then java_optional.get else Nothing
```